HW1/task1.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+print("Введите трехзначное число")
+a = int(input())
+if 100 <= a <= 999:
+    print("Сумма всех цифр числа = ", (a//100) + (a//10%10) + (a%100%10))
```

Comment:
Тут скобки не нужны, а вот пробелы наоборот.
```suggestion
    print("Сумма всех цифр числа = ", a // 100 + a // 10 % 10 + a % 100 % 10)
```